### PR TITLE
QA-436 / QA-437 / QA-438 Basic content creation sentences

### PR DESCRIPTION
**Pull request by miguelcleverti**
Auxiliary sentences used to create basic dummy content to be used for BDD testing

Related jira to Issues:
https://jira.ez.no/browse/QA-436
https://jira.ez.no/browse/QA-437
https://jira.ez.no/browse/QA-438

### CODE REVIEW

**Comment by miguelcleverti** [comment]
@andrerom @pcardiga @micszo

File: Context/Object/BasicContent.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    public function createBasicFolder( $path )
+    {
+        $names = explode( '/', $path );
+        $name = end( $names );
+        $fields = array( 'name' => $name );
+        return $this->getBasicContentManager()->createContentwithPath( $path, $fields, 'folder' );
+    }
+
+    /**
+     * @Given a/an :path article exists
```
Comment (andrerom) [inline Context/Object/BasicContent.php]: this is a bit of a nitpick maybe, but while it is kind of understandable we refer to a folder by path, it is not necessarily intuitive when referring to an article when the sentence does not refer to location.

**Comment by andrerom** [comment]
other then nitpick: +1

File: Context/Object/BasicContent.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * Sentences for Fields
+ */
+trait BasicContent
+{
+    /**
+     * @Given a/an :path folder exists
+     */
+    public function createBasicFolder( $path )
+    {
+        $names = explode( '/', $path );
```
Comment (joaoinacio) [inline Context/Object/BasicContent.php]: Small nitpick:
If for some reason `:path` ends with '/' (it shouldn't), the folder name will be empty.
For precaution `$path` could be `rtrim($path)`

**Comment by andrerom** [comment]
Miguel: ping me on skype when Joao's comment is handled and I'll merge, next time answear comments to avoid pr's dying. This is all notification based so if you don't, no one keeps track of them. So push your own pr's, no one else will normally ;)

**Comment by andrerom** [comment]
Thanks @miguelcleverti, merging.

**Pull request merged**